sparrow-py/pytests/filter_test.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import sparrow_py as kt
+
+
+@pytest.fixture(scope="module")
```

Comment:
Thoughts on grouping these?

1. Fewer separate test files / directories of golden tests
2. Ability to share fixtures between tests in a file

Perhaps:
* `time_test` for `time_of` (we'll add other time stuff)
* `collection_test` for `length`
* `logical_test` or `conditional_test` for `if` and `null_if`